docs/main/user-guides/migration/legacy-storage-to-new-storage/migration-overview.md

```diff
@@ -0,0 +1,27 @@
+---
+id: migration-overview
+title: Migration from Legacy Storage to New Storage
+keywords:
+ - Migration Overview
+description: This section outlines the process of migrating the legacy storage to new storage.
```

Comment:
'new storage' seems incorrect, need a better phrase.

Reply:
As discussed, we are using the phrase "latest storage solution"